### ISSUE TITLE
observation_registration packages not released

### ISSUE THREAD

**Issue by nilsbore** [strands-project/strands_3d_mapping#117]
@RaresAmbrus Is there a reason for this? I'm trying to release the whole repo atm.

**Comment by RaresAmbrus** [comment]
@nilsbore I guess this package hasn't been released yet (which also means a whole bunch of other packages haven't been released). Some packages in this repo haven't been released in two years (from the changelog the last release was on 0.0.12 (2015-08-21). Do you want to try to release everything?

**Comment by nilsbore** [comment]
Yeah, the observation registration folder includes a whole bunch of packages also. From my understanding, we need to release the whole repo, rather than individual packages. Do you know if there is anything standing in the way of a release?

**Comment by RaresAmbrus** [comment]
Most probably the packages are not ready for releasing in terms of their `package.xml` dependencies and probably `CMakeLists` install / library targets.

**Comment by nilsbore** [comment]
I remember something about a problem with the dependencies of the SiftGPU package. Is that correct?
